assets/prefabs/structures/fortressvillage/fortressVillageBuilding/fortressVillageCattleBarn.prefab

```diff
@@ -0,0 +1,90 @@
+{
+    "StructureTemplate": {
+        "type": "GooeysQuests:fortressVillageBuilding",
+        "spawnChance": 100
+    },
+    "SpawnBlockRegions": {
+        "regionsToFill": [
+     { "blockType": "CoreBlocks:Plank:engine:slope.RIGHT", "region": { "min": [9, 6, 0], "size": [1, 1, 20]}},
```

Comment:
Trivial: think something went wrong with indentation here (should be deeper). Might leave a bracket or two near the end out of whack too. And ideally newline at the end of every text file or GitHub shows a funny little red symbol at the end :-)